pkg/bootstrap/init.go

```diff
@@ -315,5 +315,29 @@ func deleteExistingODLM(client client.Client) error {
 		klog.Error("Failed to delete ODLM Cluster Service Version v1.2.0 in the ibm-common-services namespace")
 		return err
 	}
+
```

Comment:
I think we can get the `installedCSV` from the subscription status, if do this, we will not hard code the csv version int the code.

Reply:
I did not know this, could you create a PR about this?

Reply:
ensure that there are not too many changes on this, we are near the release, I don't want to increase the risks.